<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>dialog</class>
 <widget class="QDialog" name="dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1070</width>
    <height>776</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QStackedWidget" name="stackedWidget">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>140</y>
     <width>911</width>
     <height>591</height>
    </rect>
   </property>
   <property name="currentIndex">
    <number>1</number>
   </property>
   <widget class="QWidget" name="stack_main">
    <widget class="QLabel" name="main_lbl">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>20</y>
       <width>201</width>
       <height>81</height>
      </rect>
     </property>
     <property name="text">
      <string>main 페이지 입니다</string>
     </property>
    </widget>
    <widget class="QTextBrowser" name="main_txt">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>110</y>
       <width>256</width>
       <height>192</height>
      </rect>
     </property>
     <property name="html">
      <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'Gulim'; font-size:9pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;프로그램 소개&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="stack_app">
    <widget class="QLabel" name="app_lbl">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>20</y>
       <width>131</width>
       <height>61</height>
      </rect>
     </property>
     <property name="text">
      <string>제품등록</string>
     </property>
    </widget>
    <widget class="QComboBox" name="app_combo_type">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>90</y>
       <width>91</width>
       <height>31</height>
      </rect>
     </property>
     <item>
      <property name="text">
       <string>청소기</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>전자렌지</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>전기밥솥</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>에어컨</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>세탁기</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>tv</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>냉장고</string>
      </property>
     </item>
    </widget>
    <widget class="QLineEdit" name="app_lineedit_search">
     <property name="geometry">
      <rect>
       <x>160</x>
       <y>90</y>
       <width>191</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>모델명 검색</string>
     </property>
    </widget>
    <widget class="QPushButton" name="app_btn_search">
     <property name="geometry">
      <rect>
       <x>370</x>
       <y>90</y>
       <width>91</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>검색</string>
     </property>
    </widget>
    <widget class="QLabel" name="app_lbl_direct">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>480</y>
       <width>131</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>직접등록</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="app_lineedit_model">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>530</y>
       <width>141</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>모델명</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="app_lineedit_power">
     <property name="geometry">
      <rect>
       <x>210</x>
       <y>530</y>
       <width>131</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>소비전력</string>
     </property>
    </widget>
    <widget class="QPushButton" name="app_btn_direct_register">
     <property name="geometry">
      <rect>
       <x>700</x>
       <y>530</y>
       <width>91</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>등록</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="app_lineedit_hour">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>430</y>
       <width>131</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>사용시간</string>
     </property>
    </widget>
    <widget class="QPushButton" name="app_btn_register">
     <property name="geometry">
      <rect>
       <x>210</x>
       <y>440</y>
       <width>71</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>등록</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="app_lineedit_size">
     <property name="geometry">
      <rect>
       <x>480</x>
       <y>530</y>
       <width>81</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>용량</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="app_lineedit_direct_hour">
     <property name="geometry">
      <rect>
       <x>370</x>
       <y>530</y>
       <width>81</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>사용시간</string>
     </property>
    </widget>
    <widget class="QComboBox" name="app_combo_efficiency">
     <property name="geometry">
      <rect>
       <x>590</x>
       <y>530</y>
       <width>81</width>
       <height>31</height>
      </rect>
     </property>
     <item>
      <property name="text">
       <string>1</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>2</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>3</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>4</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>5</string>
      </property>
     </item>
    </widget>
    <widget class="QLabel" name="label_10">
     <property name="geometry">
      <rect>
       <x>590</x>
       <y>500</y>
       <width>101</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>에너지효율등급</string>
     </property>
    </widget>
    <widget class="QTableWidget" name="app_model_search">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>140</y>
       <width>541</width>
       <height>181</height>
      </rect>
     </property>
     <property name="autoScrollMargin">
      <number>16</number>
     </property>
     <property name="iconSize">
      <size>
       <width>0</width>
       <height>1</height>
      </size>
     </property>
     <attribute name="verticalHeaderVisible">
      <bool>false</bool>
     </attribute>
     <column>
      <property name="text">
       <string>모델명</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>소비전력</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>용량</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>등급</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>탄소인증</string>
      </property>
     </column>
    </widget>
   </widget>
   <widget class="QWidget" name="stack_result">
    <widget class="QLabel" name="result_lbl">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>0</y>
       <width>141</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>결과 페이지</string>
     </property>
    </widget>
    <widget class="QTabWidget" name="result_tab_visulal">
     <property name="geometry">
      <rect>
       <x>620</x>
       <y>100</y>
       <width>261</width>
       <height>341</height>
      </rect>
     </property>
     <property name="currentIndex">
      <number>1</number>
     </property>
     <widget class="QWidget" name="day">
      <attribute name="title">
       <string>day</string>
      </attribute>
     </widget>
     <widget class="QWidget" name="month">
      <attribute name="title">
       <string>month</string>
      </attribute>
     </widget>
    </widget>
    <widget class="QLabel" name="result_lbl_visual">
     <property name="geometry">
      <rect>
       <x>650</x>
       <y>40</y>
       <width>191</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>일별 월별 사용 동적시각화</string>
     </property>
    </widget>
    <widget class="QTableWidget" name="result_table_power">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>110</y>
       <width>421</width>
       <height>181</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="result_lbl_recommend">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>310</y>
       <width>271</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>추천 제품 5개/ 탄소발자국 인증 제품 5개</string>
     </property>
    </widget>
    <widget class="QTableWidget" name="result_table_recommend">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>360</y>
       <width>421</width>
       <height>181</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="result_lbl_power">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>60</y>
       <width>341</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>보유제품 소비전력&amp;소비전력 높은 제품 행 색깔 바꿔 알려주기</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="stack_my">
    <widget class="QLabel" name="my_lbl">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>20</y>
       <width>191</width>
       <height>61</height>
      </rect>
     </property>
     <property name="text">
      <string>마이페이지</string>
     </property>
    </widget>
    <widget class="QLabel" name="my_lbl_app">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>80</y>
       <width>151</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>가전 제품 보유내역</string>
     </property>
    </widget>
    <widget class="QTableWidget" name="my_table_app">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>120</y>
       <width>501</width>
       <height>192</height>
      </rect>
     </property>
    </widget>
    <widget class="QTextBrowser" name="my_txt_">
     <property name="geometry">
      <rect>
       <x>610</x>
       <y>230</y>
       <width>241</width>
       <height>231</height>
      </rect>
     </property>
     <property name="html">
      <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'Gulim'; font-size:9pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;회원의 전기 사용량에 대한 설명&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
    </widget>
    <widget class="QTextBrowser" name="my_table_graph">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>350</y>
       <width>501</width>
       <height>192</height>
      </rect>
     </property>
     <property name="html">
      <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'Gulim'; font-size:9pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;에너지 사용 세부 조회- &lt;span style=&quot; font-family:'맑은 고딕';&quot;&gt;에너지 사용량을 일별, 월별, 분기별로 원형그래프를 이용하여 전체 에너지 사용량 대비 각 부분의 소비 비율을 파악할 수 있도록 하며, 수치를 기재 하여 세부 데이터를 확인할 수 있도록 함.&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
    </widget>
    <widget class="QPushButton" name="my_btn_update">
     <property name="geometry">
      <rect>
       <x>660</x>
       <y>20</y>
       <width>101</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>개인정보 수정</string>
     </property>
    </widget>
    <widget class="QPushButton" name="my_btn_delete">
     <property name="geometry">
      <rect>
       <x>780</x>
       <y>20</y>
       <width>101</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>회원탈퇴</string>
     </property>
    </widget>
    <widget class="QTextBrowser" name="my_txt_point">
     <property name="geometry">
      <rect>
       <x>610</x>
       <y>120</y>
       <width>241</width>
       <height>81</height>
      </rect>
     </property>
     <property name="html">
      <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'Gulim'; font-size:9pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot; margin-top:12px; margin-bottom:12px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-family:'맑은 고딕';&quot;&gt;포인트 적립/사용내역 조회- (포인트 버튼 있는 곳에 있긴 함.)&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="stack_point">
    <widget class="QLabel" name="point_lbl">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>20</y>
       <width>161</width>
       <height>51</height>
      </rect>
     </property>
     <property name="text">
      <string>포인트 페이지 입니다.</string>
     </property>
    </widget>
    <widget class="QTableWidget" name="point_table">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>70</y>
       <width>411</width>
       <height>171</height>
      </rect>
     </property>
     <column>
      <property name="text">
       <string>회원번호</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>아이디</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>적립내역</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>사용내역</string>
      </property>
     </column>
    </widget>
    <widget class="QPushButton" name="point_btn_charity">
     <property name="geometry">
      <rect>
       <x>310</x>
       <y>260</y>
       <width>121</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>기부하기</string>
     </property>
    </widget>
    <widget class="QPushButton" name="point_btn__use">
     <property name="geometry">
      <rect>
       <x>160</x>
       <y>260</y>
       <width>121</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>사용하기</string>
     </property>
    </widget>
    <widget class="QTextBrowser" name="point_txt">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>340</y>
       <width>551</width>
       <height>192</height>
      </rect>
     </property>
     <property name="html">
      <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'Gulim'; font-size:9pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;- 나무를 심어 온실가스 절감이 가능하다는 문구 예전에 얘기했던 거 일단 넣었습니다&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QPushButton" name="btn_my">
   <property name="geometry">
    <rect>
     <x>670</x>
     <y>30</y>
     <width>141</width>
     <height>81</height>
    </rect>
   </property>
   <property name="text">
    <string>MyPage</string>
   </property>
   <property name="checkable">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_main">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>30</y>
     <width>141</width>
     <height>81</height>
    </rect>
   </property>
   <property name="text">
    <string>MainPage</string>
   </property>
   <property name="checkable">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_result">
   <property name="geometry">
    <rect>
     <x>470</x>
     <y>30</y>
     <width>141</width>
     <height>81</height>
    </rect>
   </property>
   <property name="text">
    <string>결과</string>
   </property>
   <property name="checkable">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_point">
   <property name="geometry">
    <rect>
     <x>870</x>
     <y>30</y>
     <width>141</width>
     <height>81</height>
    </rect>
   </property>
   <property name="text">
    <string>포인트</string>
   </property>
   <property name="checkable">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_app">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>30</y>
     <width>141</width>
     <height>81</height>
    </rect>
   </property>
   <property name="text">
    <string>제품등록</string>
   </property>
   <property name="checkable">
    <bool>true</bool>
   </property>
  </widget>
  <zorder>btn_main</zorder>
  <zorder>stackedWidget</zorder>
  <zorder>btn_my</zorder>
  <zorder>btn_result</zorder>
  <zorder>btn_point</zorder>
  <zorder>btn_app</zorder>
 </widget>
 <resources/>
 <connections/>
</ui>
